src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -248,18 +259,23 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
       </div>
     );
   }
-  private async findMappedNodes(): Promise<sdk.AssetMapping[]> {
+
+  private async findMappedNodes(): Promise<AssetMapping3D[]> {
     const { assetId, modelId, revisionId } = this.props;
 
     if (!assetId) {
       return Promise.resolve([]);
     }
 
-    const { items } = await sdk.ThreeD.listAssetMappings(modelId, revisionId, {
+    return await this.context!.assetMappings3D.list(modelId, revisionId, {
       assetId,
-    });
+    }).autoPagingToArray();
+  }
+
+  private async getRevision() {
```

Comment:
no need in async here since the function returns a promise

Reply:
true, fixed